sdk/storage/Azure.Storage.Webjobs.Extensions.Blobs/tests/Azure.WebJobs.Extensions.Storage.Blobs.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup>
+    <Version>1.0.0-beta.1</Version>
+    <Description>This extension adds bindings for Storage</Description>
+    <IsPackable>false</IsPackable>
+    <PublicSign>false</PublicSign>
+    <DelaySign>true</DelaySign>
+    <AssemblyOriginatorKeyFile>PublicKey.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="Moq" />
+    <PackageReference Include="xunit" />
```

Comment:
Do you plan to keep xunit or eventually switch to NUnit?

Reply:
Yes. we have work item for that. Which is also prerequisite to have proper integration of live tests with pipelines.